.github/workflows/stale.yml

```diff
@@ -0,0 +1,70 @@
+name: Stale issues and pull-requests
+on:
+  schedule:
+    - cron: 0 * * * *
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Idle number of days before marking issues/prs stale.
+          days-before-stale: 1000
+
+          # Idle number of days before closing stale issues/prs.
+          days-before-close: 28
+
+          # Comment on the staled issues.
+          stale-issue-message: |
+            In order to maintain a backlog of relevant issues, we automatically label them as stale after 1,000 days of inactivity.
+
+            If this issue is still important to you, then please comment on this issue or remove the stale label.
+
+            Otherwise this issue will be automatically closed in 28 days time.
+
+          # Comment on the staled prs.
+          stale-pr-message: | 
+            In order to maintain a backlog of relevant PRs, we automatically label them as stale after 1,000 days of inactivity.
+
+            If this PR is still important to you, then please comment on this PR or remove the stale label.
+
+            Otherwise this PR will be automatically closed in 28 days time.
+
+          # Comment on the staled issues while closed.
+          close-issue-message: |
+            This stale issue has been automatically closed due to a lack of community activity.
+            
+            If you still care about this issue, then please either:
+              * Re-open this issue, if you have sufficient permissions, or
+              * Add a comment pinging `@SciTools/iris-devs` who will re-open on your behalf.
```

Comment:
To be honest, we could have a GHA `bot` that auto-opens closed issues/prs whenever the user adds a comment like `@SciTools/iris-devs, please re-open` akin to `@conda-forge-admin, please rerender` on a `conda-forge` feedstock.